src/components/table.ts

```diff
@@ -9,7 +9,7 @@ export module Components {
     wantsHeightArr: boolean[];
   }
 
-  export type _IterateLayoutResult = {
+  type _IterateLayoutResult = {
```

Comment:
The type is being used to encapsulate the large object type for an instance variable.  We should actually probably keep the type.